crates/api/src/instance.rs

```diff
@@ -100,14 +101,27 @@ impl Instance {
         })
     }
 
-    pub fn exports(&self) -> &[Extern] {
-        &self.exports
+    /// Returns the associated [`Store`] that this `Instance` is compiled into.
+    ///
+    /// This is the [`Store`] that generally serves as a sort of global cache
+    /// for various instance-related things.
+    pub fn store(&self) -> &Store {
+        self.module.store()
     }
 
+    /// Returns the associated [`Module`] that this `Instance` instantiated.
+    ///
+    /// The corresponding [`Module`] here is a static version of this `Instance`
+    /// which can be used to learn information such as naming information about
+    /// various functions.
     pub fn module(&self) -> &Module {
         &self.module
     }
 
+    pub fn exports(&self) -> &[Extern] {
```

Comment:
Nit: does this need a doc comment?

Sure, there's a number of existing methods that don't have one, but might be worth adding since the other two new functions do.

Reply:
It does, yeah, I plan on a follow-up PR to document everything in `instance.rs`